doc/modules/decomposition.rst

```diff
@@ -726,3 +735,8 @@ the data.
       matrix factorization"
       <http://scgroup.hpclab.ceid.upatras.gr/faculty/stratis/Papers/HPCLAB020107.pdf>`_
       C. Boutsidis, E. Gallopoulos, 2008
+
+    * `"Fast coordinate descent methods with variable selection for
+      non-negative matrix factorization."
+      <http://apps.cs.utexas.edu/tech_reports/reports/tr/TR-2017.pdf>`_
```

Comment:
I would rather remove this reference since our implementation is a mere application of CD to the NMF objective. Once we add GKL-diverge, we can add it back.